backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Tests/Controllers/SingleRightCotrollerTest.cs

```diff
@@ -0,0 +1,90 @@
+using System.Net;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Text.Json;
+using Altinn.AccessManagement.UI.Controllers;
+using Altinn.AccessManagement.UI.Core.Models.SingleRight.CheckDelegationAccess;
+using Altinn.AccessManagement.UI.Core.Models.SingleRight.CheckDelegationAccess.CheckDelegationAccessDto;
+using Altinn.AccessManagement.UI.Mocks.Utils;
+using Altinn.AccessManagement.UI.Tests.Utils;
+
+namespace Altinn.AccessManagement.UI.Tests.Controllers
+{
+    /// <summary>
+    ///     Test class for <see cref="SingleRightCotrollerTest"></see>
+    /// </summary>
+    [Collection("SingleRightCotrollerTest")]
+    public class SingleRightCotrollerTest : IClassFixture<CustomWebApplicationFactory<SingleRightController>>
+    {
+        private readonly HttpClient _client;
+        private readonly CustomWebApplicationFactory<SingleRightController> _factory;
+        /// <summary>
+        ///     Constructor setting up factory, test client and dependencies
+        /// </summary>
+        /// <param name="factory">CustomWebApplicationFactory</param>
+        public SingleRightCotrollerTest(CustomWebApplicationFactory<SingleRightController> factory)
+        {
+            _factory = factory;
+            _client = SetupUtils.GetSingleRightTestClient(factory);
+            _client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+            string token = PrincipalUtil.GetAccessToken("sbl.authorization");
+            _client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", token);
+        }
+
+        [Fact]
+        public async Task DelegationAccessCheck_valid_response()
+        {
+            // Arrange
+            string partyId = "50004223";
```

Comment:
I'm guessing you're going to need to add this once you enable authorization on your controller

```suggestion
            string partyId = "50004223";
            var token = PrincipalUtil.GetToken(1337, 501337, 2);
            _client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", token);
```

Reply:
I need it for the route as well